smartling-job-batches-api/src/main/java/com/smartling/api/jobbatches/v2/pto/CreateJobRequestPTO.java

```diff
@@ -0,0 +1,27 @@
+package com.smartling.api.jobbatches.v2.pto;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+public class CreateJobRequestPTO
+{
+    private String nameTemplate;
```

Comment:
Could you pls add a validation that it's not empty?

Reply:
We have the validation in the endpoint itself. Normally we don't have any client side validations in this SDK.